src/test/java/net/sf/jabref/logic/importer/fileformat/BibtexParserTest.java

```diff
@@ -127,15 +127,6 @@ public void singleFromStringReturnsNullIfNoEntryRecognized() {
     }
 
     @Test
-    public void parseTwoTimesReturnsSameResult() throws IOException {
```

Comment:
Why can't an equivalent test be written using the new structure? Two times calling the parser with the string thing and then compare the resulting entries?